src/main/resources/gdb/target.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0"?>
```

Comment:
Would name this file such that it reflects the target arch as well, `target-riscv64` e.g.?